diff --git a/docroot/sites/all/modules/contrib/text_resize/text_resize.module b/docroot/sites/all/modules/contrib/text_resize/text_resize.module
index 7ead9a4..6cade78 100644
--- a/docroot/sites/all/modules/contrib/text_resize/text_resize.module
+++ b/docroot/sites/all/modules/contrib/text_resize/text_resize.module
@@ -89,7 +89,7 @@ function text_resize_block_info() {
   // Makes it show up on /admin/build/block page.
   $blocks[0] = array(
     'info' => t('Text Resize'),
-    'cache' => DRUPAL_CACHE_PER_PAGE,
+    'cache' => DRUPAL_NO_CACHE,
   );
   return $blocks;
 }
